authoring-map/README.md

```diff
@@ -4,6 +4,8 @@ This sample demonstrates how to author a map and save it to your portal
 ![screenshot1](https://cloud.githubusercontent.com/assets/12448081/16025921/69544758-317f-11e6-9417-de4d561064bf.png) ![screenshot2](https://cloud.githubusercontent.com/assets/12448081/16025943/89f2ff7c-317f-11e6-9063-82dd7139458e.png)
 ![screenshot3](https://cloud.githubusercontent.com/assets/12448081/16026079/93dbfaa6-3180-11e6-9a89-9e1513dff881.png) ![screenshot4](https://cloud.githubusercontent.com/assets/12448081/16026092/a463059a-3180-11e6-84a6-6814515f3287.png)
 #How to use the sample
+**Note**
+Before you use the sample, register your application on [developers](https://developers.arcgis.com/) site and make a note of your client-id, URL path and create a redirect URI. Goto **res>values>portal_settings.xml** and enter your account URL path, client-id and redirect-uri. 
```

Comment:
@manishk3189 Make this more of a step by step process, e.g.:

1. Register your app on developers site (try to find a better [link](https://developers.arcgis.com/authentication/signing-in-arcgis-online-users/) that explains what register means)
2. Locate your client id ... 

Essentially, walk the developer through the process in steps.